roles/edpm_reboot/defaults/main.yaml

```diff
@@ -0,0 +1,21 @@
+---
+# Copyright 2023 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# All variables intended for modification should be placed in this file.
+
+edpm_reboot_workload_protection: true
+_no_reboot_message: "Reboot should not be necessary."
```

Comment:
For coherence it should be something like `edpm_reboot_not_necessary_message`

Reply:
edpm_reboot_not_necessary_message removed